protocol-models/bin/generate-python-dataclasses.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+git config --global --add safe.directory /airbyte-protocol
+
+ROOT_DIR=${ROOT_DIR:-$(git rev-parse --show-toplevel)}
+
+[ -z "$ROOT_DIR" ] && exit 1
+
+
+
+YAML_DIR=protocol-models/src/main/resources/airbyte_protocol
+OUTPUT_DIR=protocol-models/python/airbyte_protocol_dataclasses/airbyte_protocol_dataclasses/models
+
+python -m pip install --upgrade pip
+pip install -U datamodel_code_generator
+
+rm -rf "$ROOT_DIR/$OUTPUT_DIR"/*.py
+mkdir -p "$ROOT_DIR/$OUTPUT_DIR"
+
+echo "# generated by generate-python-dataclasses" > "$ROOT_DIR/$OUTPUT_DIR"/__init__.py
+echo "name = 'models'" >> "$ROOT_DIR/$OUTPUT_DIR"/__init__.py
+
+for f in "$ROOT_DIR/$YAML_DIR"/*.yaml; do
+  filename_wo_ext=$(basename "$f" | cut -d . -f 1)
+  echo "from .$filename_wo_ext import *" >> "$ROOT_DIR/$OUTPUT_DIR"/__init__.py
+
+  datamodel-codegen \
+    --input "$ROOT_DIR/$YAML_DIR/$filename_wo_ext.yaml" \
+    --output "$ROOT_DIR/$OUTPUT_DIR/$filename_wo_ext.py" \
+    --output-model-type dataclasses.dataclass \
+    --target-python-version 3.10 \
```

Comment:
should we target 3.8 since that's the minimum version?

Reply:
Leave 3.8 support sincee 3.8 and 3.10 version are identical.